### PR TITLE
Update Yandex.Market screenshot and docs (added cost multiplier)

### PR DESCRIPTION
**Pull request by MaksimKiselev**
https://app.asana.com/0/1001628190565958/1134734512512639/f

### CODE REVIEW

File: _docs/datasources/yandex market.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ After authorization you need to set the data source parameters.
 
 (6) If normalized costs report enabled, you can select billing currency of report.
 
-(7) **Import offer statistics report**  - enable offer statistics report import.
+(7) If normalized costs report enabled, you can set value multiplier. Multiplied by the cost in the report.
```
Comment (ArtemBogomolov) [inline _docs/datasources/yandex market.md]:
```suggestion
(7) If the normalized costs report is enabled, you can set the value multiplier. Multiplied by the cost in the report.
```

File: _docs/datasources/yandex market.md [hunk 2]
```diff
@@ -31,7 +31,9 @@ After authorization you need to set the data source parameters.
 
 (6) If normalized costs report enabled, you can select billing currency of report.
```
Comment (ArtemBogomolov) [inline _docs/datasources/yandex market.md]:
```suggestion
(6) If the normalized costs report is enabled, you can select the billing currency of the report.
```

File: _docs/datasources/yandex market.md (hunk 1 above)
Comment (MaksimKiselev) [reply _docs/datasources/yandex market.md]: Thx

File: _docs/datasources/yandex market.md (hunk 2 above)
Comment (MaksimKiselev) [reply _docs/datasources/yandex market.md]: Thx

**Pull request merged**